package.json

```diff
@@ -26,5 +26,8 @@
   "homepage": "https://github.com/riyacchi/eris-pagination#readme",
   "dependencies": {
     "eris-reactions": "^0.1.2"
+  },
+  "devDependencies": {
+    "eris": "^0.10.1"
   }
```

Comment:
Not strictly necessary, but Typescript docs say it's advisable to add the `"types"` key even when the typing file is `index.d.ts.`.